pkg/cmd/provisioning/alibabacloud/attach-ram-policy.go

```diff
@@ -0,0 +1,293 @@
+package alibabacloud
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/auth/credentials/provider"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/ram"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/sts"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/cloud-credential-operator/pkg/alibabacloud"
+	credreqv1 "github.com/openshift/cloud-credential-operator/pkg/apis/cloudcredential/v1"
+	"github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+const (
+	secretManifestsTemplate = `apiVersion: v1
+stringData:
+  credentials: |-
+    [default]
+    type = access_key
+    access_key_id: %s
+    access_key_secret: %s
```

Comment:
I am not sure if those `:` will work on `ini` files. @kwoodson and I was testing and we needed to change to `=` to make it work.